Xamarin.Forms.Platform.WPF/Renderers/ScrollViewRenderer.cs

```diff
@@ -156,14 +164,14 @@ void UpdateMargins()
 		void UpdateOrientation()
```

Comment:
Won't this just throw away the scroll bar visibility set explicitly? Ie reorientation won't respect the enum

Reply:
I'll take a look at it again, I'll test changing the orientation and see how it affects things.

What about the case when you have the scrolling horizontal with the bar shown "Always" then switch to vertical.  Hide the bar like the current code does or honor the visibility  setting?